### DISCUSSION TITLE
♥ the "Forecast Stock Market Data using Time Series" YouTube video

### DISCUSSION THREAD

**Discussion by PureKrome** [ravendb/ravendb#13466] [General]
Just some positive feedback re: https://ayende.com/blog/196129-C/working-with-market-data-using-time-series-in-ravendb?Key=1260ce88-f3fb-4493-aabe-dbc99a662491

I've never done time series. I just watched the YT video on it with RavenDb and it's really lovely and explained great! Well done! Love seeing things like this, also backed up with a sample app / code / etc ... 

These things will definitely help with getting people into using this product, hopefully 🙏🏻 

Well done! (The Rock, approves!)

![](https://media.giphy.com/media/QLvRBqfLXCphu/giphy.gif)

**Comment by ayende** [comment]
Very happy to hear that!

**Comment by kamranayub** [comment]
Thanks, put a lot of work into that! 🙏
